### PR TITLE
[ja] Translate data plane definition

### LINKED ISSUE #46446: [ja] Translate content/en/docs/reference/glossary/data-plane.md into Japanese
_Note that this issue is for Japanese Speaker_

**This is a Feature Request**

**What would you like to be added**

Translate `content/en/docs/reference/glossary/data-plane.md` into Japanese

**Why is this needed**

There is no Japanese translation for this page

**Comments**

- File to translate: 
    - https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/data-plane.md
- Original:
    - https://kubernetes.io/ja/docs/reference/glossary/?all=true#term-data-plane

日本語に翻訳する際は、以下のドキュメントを参考にしてください。

- Japanese Localization document: https://kubernetes.io/ja/docs/contribute/localization/

---

/area localization
/language ja
/good-first-issue

### PR DESCRIPTION
**Pull request by tamilselvan1102**
Fixes #46446

### CODE REVIEW

**Comment by tamilselvan1102** [comment]
cc @Okabe-Junya

**Comment by tamilselvan1102** [comment]
preview page: https://deploy-preview-46555--kubernetes-io-main-staging.netlify.app/ja/docs/reference/glossary/?fundamental=true#term-data-plane

File: content/ja/docs/reference/glossary/data-plane.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+date: 2019-05-12
+full_link:
+short_description: >
+  コンテナが実行してネットワークに接続できるように、CPU、メモリー、ネットワーク、ストレージなどの容量を提供するレイヤーです。
+
+aka:
+tags:
+- fundamental
+---
+ コンテナが実行してネットワークに接続できるように、CPU、メモリー、ネットワーク、ストレージなどの容量を提供するレイヤーです。
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/data-plane.md:13]:
```suggestion
  コンテナを実行し、ネットワークに接続できるように、CPU、メモリ、ネットワーク、ストレージなどの容量を提供するレイヤーです。
```

ditto

**Review by Okabe-Junya — CHANGES_REQUESTED** [review]
Thank you for your contribution!!

I made a few comments, PTAL :)

**Review by Okabe-Junya — APPROVED** [review]
LGTM!! Thank you!

/lgtm
/assign inductor

File: content/ja/docs/reference/glossary/data-plane.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: データプレーン
+id: data-plane
+date: 2019-05-12
+full_link:
+short_description: >
+  コンテナを実行し、ネットワークに接続できるように、CPU、メモリ、ネットワーク、ストレージなどの容量を提供するレイヤーです。
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/data-plane.md:7]:
```suggestion
  コンテナを実行し、ネットワークに接続できるように、CPU、メモリ、ネットワーク、ストレージなどのキャパシティを提供するレイヤーです。
```

File: content/ja/docs/reference/glossary/data-plane.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+aka:
+tags:
+- fundamental
+---
+  コンテナを実行し、ネットワークに接続できるように、CPU、メモリ、ネットワーク、ストレージなどの容量を提供するレイヤーです。
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/data-plane.md:13]:
```suggestion
  コンテナを実行し、ネットワークに接続できるように、CPU、メモリ、ネットワーク、ストレージなどのキャパシティを提供するレイヤーです。
```

**Review by Okabe-Junya — CHANGES_REQUESTED** [review]
@tamilselvan1102 

Sorry for the inconvenience, but I have further reviewed this translation and would like to request a few simple fixes. 

(Note that this is a part of the glossary, so a very high-quality translation is required.)

/lgtm cancel

**Review by Okabe-Junya — APPROVED** [review]
Thank you!!

/lgtm

**Review by inductor — APPROVED** [review]

**Pull request merged**